fe/fe-core/src/main/java/com/starrocks/connector/unified/UnifiedConnector.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.unified;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import com.starrocks.catalog.Table;
+import com.starrocks.connector.Connector;
+import com.starrocks.connector.ConnectorContext;
+import com.starrocks.connector.ConnectorMetadata;
+import com.starrocks.connector.config.ConnectorConfig;
+import com.starrocks.connector.delta.DeltaLakeConnector;
+import com.starrocks.connector.hive.HiveConnector;
+import com.starrocks.connector.hudi.HudiConnector;
+import com.starrocks.connector.iceberg.IcebergConnector;
+import com.starrocks.credential.CloudConfiguration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.starrocks.catalog.Table.TableType.DELTALAKE;
+import static com.starrocks.catalog.Table.TableType.HIVE;
+import static com.starrocks.catalog.Table.TableType.HUDI;
+import static com.starrocks.catalog.Table.TableType.ICEBERG;
+
+public class UnifiedConnector implements Connector {
+    public static final String HIVE_METASTORE_URIS = "hive.metastore.uris";
+    public static final String HIVE_METASTORE_TYPE = "hive.metastore.type";
+    public static final List<String> SUPPORTED_METASTORE_TYPE = Lists.newArrayList("hive", "glue", "dlf");
+    private final Map<String, String> properties;
+    private final String catalogName;
+    private final Map<Table.TableType, Connector> connectorMap;
+
+    public UnifiedConnector(ConnectorContext context) {
+        this.properties = context.getProperties();
+        this.catalogName = context.getCatalogName();
+        this.connectorMap = ImmutableMap.of(
+                HIVE, new HiveConnector(context),
+                ICEBERG, new IcebergConnector(context),
```

Comment:
How to successfully create a unified catalog if you have not set the iceberg parameter.

Reply:
use `unified.metastore.type`